test/routes/definitions.js

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) The Linux Foundation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const { ExponentialRetryPolicyFilter } = require('azure-storage')
+const { expect } = require('chai')
+const httpMocks = require('node-mocks-http')
+const sinon = require('sinon')
+const definitionsRoutes = require('../../routes/definitions')
+
+describe('Definition route', () => {
+  it('accepts a good GET request', async () => {
+    const request = createGetRequest()
+    const response = httpMocks.createResponse()
+    const definitionService = createDefinitionService()
+    const router = createRoutes(definitionService)
+    await router._getDefinition(request, response)
+    expect(response.statusCode).to.be.eq(200)
+    //    const getDefinitionSpy = definitionService.get
```

Comment:
Could we remove these commented out lines?

Reply:
Gah! Good catch!

Reply:
That's what PR reviews are for, haha. Happens to everyone.